Dockerfile

```diff
@@ -2,9 +2,9 @@ FROM fedora
 
 RUN dnf install -y git dnf-utils gcc meson ninja-build libseccomp-static libcap-static \
     make python git gcc automake autoconf libcap-devel systemd-devel yajl-devel libseccomp-devel cmake \
-    go-md2man glibc-static python3-libmount libtool \
+    go-md2man glibc-static python3-libmount libtool pcre2-static \
     && yum-builddep -y systemd \
     && git clone --depth 1 https://github.com/systemd/systemd.git \
-    && (mkdir systemd/build; cd systemd/build; meson ..; ninja version.h; ninja libsystemd.a; cp libsystemd.a /usr/lib64) \
+    && (mkdir systemd/build; cd systemd/build; meson -Dstatic-libsystemd=true -Dselinux=false ..; ninja version.h; ninja libsystemd.a; cp libsystemd.a /usr/lib64) \
     && (git clone --depth=1 https://github.com/lloyd/yajl.git; cd yajl; ./configure LDFLAGS=-static; cd build; make -j $(nproc); find . -name '*.a' -exec cp \{\} /usr/lib64 \;) \
-    && (cd /crun; make distclean; ./configure CFLAGS="-O3" LDFLAGS="-static-libgcc -static" LIBS="/usr/lib64/libcap.a /usr/lib64/libseccomp.a /usr/lib64/libsystemd.a /usr/lib64/libyajl_s.a"; make -j $(nproc))
+    && (cd /crun;  ./configure CFLAGS="-O3" CRUN_LDFLAGS=-all-static LDFLAGS="-static-libgcc -static" LIBS="/usr/lib64/libsystemd.a /usr/lib64/libyajl_s.a /usr/lib64/libcap.a /usr/lib64/libseccomp.a /usr/lib64/libpcre2-8.a /usr/lib64/librt.a /usr/lib64/libpthread.a /usr/lib64/libc.a"; make -j $(nproc))
```

Comment:
Can we use `COPY . /crun` instead of bind-mounting, which is not supported in Docker?

Reply:
sure, how do we copy back the crun executable?

Reply:
```dockerfile
FROM ...
...

FROM scratch
COPY --from=0 /crun/crun .
```

On Podman:
```console
$ podman build -t foo .
$ podman create --name foo foo
$ mkdir out
$ podman cp foo:/crun out
$ podman rm foo
```

On Docker:
```console
$ mkdir out
$ docker build --output out .
```

Reply:
thanks, pushed a new version with your suggestion